docs/environment-variables.md

```diff
@@ -12,6 +12,17 @@ With `useDotenv: true` set in your `serverless.yml` file, framework automaticall
 
 The framework looks for `.env` and `.env.{stage}` files in service directory and then tries to load them using `dotenv`. If `.env.{stage}` is found, `.env` will not be loaded. If stage is not explicitly defined, it defaults to `dev`.
 
+### Variable expansion
```

Comment:
Let's also mention what specific library is being used to perform this expansion